test/utils/types.go

```diff
@@ -0,0 +1,14 @@
+package utils
```

Comment:
`/test` [appears to mean](https://github.com/golang-standards/project-layout#test) external test apps. Seems like stuff that should only be shared between tests (and even that's arguable, as you won't be able to ever modify these without refactoring all the tests - a major pain caused by unnecessary coupling of all the tests).

Reply:
Also not really utils at all... These are common types :)